src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -44,6 +44,8 @@ internal class GuidesValidationMethods
 
         private static NodeModel lastCreatedNode;
 
+        private static Window PackageManagerWindow;
```

Comment:
I would maybe not name this so specific but use a general name for reuse by other guide?

Reply:
Maybe `CurrentGuideWindow`?